lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -948,6 +958,11 @@ sub update_status {
 
     if ($self->worker) {
         $self->worker->set_property("INTERACTIVE", $status->{status}->{interactive} // 0);
+        # mark the worker as alive
+        $self->worker->seen;
```

Comment:
I understand the reason for your change. I don't understand problem @okurz is describing.